### DISCUSSION TITLE
transmit camera

### DISCUSSION THREAD

**Discussion by barraviera** [pedroSG94/RootEncoder#1078] [General]
is it possible to transmit camera without a preview?

**Comment by pedroSG94** [comment]
Hello,

You can do it with the constructor that require context.
You have an example here:
https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/tree/master/app/src/main/java/com/pedro/rtpstreamer/backgroundexample
In this case the stream is without preview but we are adding preview dinamically. You only need remove method setView.

**Reply by barraviera** [reply]
perfect, just what I needed! Congratulations.
